addons/frag/XEH_preInit.sqf

```diff
@@ -6,20 +6,23 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
-GVAR(blackList) = [];
-GVAR(traceFrags) = false;
+GVAR(spallMaterialCache) = createHashMap;
+GVAR(spallInfoCache) = createHashMap;
+GVAR(shouldSpallCache) = createHashMap;
+GVAR(nextSpallAllowTime) = -2;
```

Comment:
It should work for -1, there was a reason to prefer -2 at one point but not anymore. As someone who didn't write it, is it something that confused you seeing it as -2 instead of -1? I can definitely change it to prevent future confusion, but I don't think the values as long as it's greater the hold of definitions and isn't confusing.

Reply:
-2 to me is oddly specific, whereas -1 means it's a default value, for me at least. Change to -1 then.

Reply:
Done